lib/tasks/translation.rake

```diff
@@ -85,17 +85,19 @@ namespace :translation do
 
   desc "Add missing translations"
   task(:add_missing, [:locale] => [:environment]) do |t, args|
-    locale = args[:locale] || "en"
+    locale = args[:locale]
     require 'i18n/tasks/cli'
 
-    I18n::Tasks::CLI.start(["add-missing", "-l", locale])
+    i18n_args = ["add-missing", "--nil-value"]
+    i18n_args << ["-l", locale] if locale
+    I18n::Tasks::CLI.start(i18n_args.flatten)
   end
 
   desc "Normalize translations"
   task(:normalize, [:locale] => [:environment]) do |t, args|
-    locale = args[:locale] || "en"
+    locale = args[:locale]
     require 'i18n/tasks/cli'
 
-    I18n::Tasks::CLI.start(["normalize", "-l", locale])
+    locale ? I18n::Tasks::CLI.start(["normalize", "-l", locale]) : I18n::Tasks::CLI.start(["normalize"])
```

Comment:
For consistency with the other rake task, I think I'd follow the same `i18n_args << ["-l", locale] if locale` approach here too.

OR, if you prefer, you could maybe make a helper method so that we can just call like this?

```ruby
I18n::Tasks::CLI.start(with_optional_locale(["normalize"]))
```